packages/render-fidelity-tools/src/components/rendering-scenario.ts

```diff
@@ -39,27 +41,67 @@ export class RenderingScenario extends LitElement {
     return `./results/${this.name}`;
   }
 
+  updated(changedProperties: Map<any, any>) {
+    super.updated(changedProperties);
+
+    if (this.analysis == null) {
+      this.loadAnalysis();
+    }
+  }
+
```

Comment:
I tried load ayalysis in constructor because it's easier to people to understand that way, but somehow it didn't work. So i just followed the same pattern as you did in image-comparision-app.ts